_posts/2018-09-20-rails-security.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "Step up the security of your Rails app | Part 1"
+date: 2018-09-20 12:10:00
+categories: ["rails", "security"]
+author: "lubc"
+---
+
+The internet is a wonderful place, but there will always be people that don't have good intentions when they visit our websites. That's why you need to be aware of the vulnerabilities that your application can have and how to avoid them. In this article I'll cover two common security problems in [Rails](https://rubyonrails.org) applications (I'll probably make a second part since this is a very extensive topic).
+
+<!--more-->
+
+It's also worth to mention that Rails itself has improved a lot [over the years](https://www.youtube.com/watch?v=Btrmc1wO3pc) to make everything more secure and easy for the people who use it. That's one of the reasons why it's so important to [upgrade your Rails project](https://www.ombulabs.com/blog/tags/upgrades).
+
+Before we start you should know that Rails has an [official security guide](https://guides.rubyonrails.org/security.html) with a large variety of common vulnerabilities. I recommend to take a look at it if you want to dive deeper into this topic. Also at the end of this article I'll leave a few links to some useful tools that will help you to level up your security in Rails.
```

Comment:
@lubc I think it sounds better if you say: "will help you level up security in your Rails application."